FWCore/ParameterSet/python/Config.py

```diff
@@ -533,11 +533,17 @@ def extend(self,other,items=()):
                 if not item.hasLabel_() :
                     item.setLabel(name)
             elif isinstance(item,Schedule):
-                self.__setattr__(name,item)
```

Comment:
If you did
```
process.extendFromProcess(f1)
process.extendFromProcess(f2)
```
Which internally used the ```ProcessWrapper``` then you could do something special for ```Schedule``` in that call.